<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inter_ticket</class>
 <widget class="QMainWindow" name="inter_ticket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>841</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>guichet</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de ticket :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_ajout_guichet">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbticket_ajout_guichet">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="etat_ajout_guichet">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>120</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Ouvert</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fermer</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="ajout_guichet">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QPushButton" name="modif_guichet">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>162</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_modif_guichet">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="etat_modif_guichet">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>106</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Ouvert</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fermer</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de ticket :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbticket_modif_guichet">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>78</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>53</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>24</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="id_modif_guichet">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>20</y>
        <width>281</width>
        <height>201</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
      <widget class="QPushButton" name="supp_guichet">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>160</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre de ticket :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>53</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>24</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="id_supp_guichet">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="nom_supp_guichet">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="nbticket_supp_guichet">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>80</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="etat_supp_guichet">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_guichet">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>791</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>240</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>240</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Par :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tri_guichet">
      <property name="geometry">
       <rect>
        <x>671</x>
        <y>240</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche_guichet">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="PDFbuttton">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>232</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimer &amp; PDF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ticket</string>
     </attribute>
     <widget class="QLineEdit" name="recherche_ticket">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>10</y>
        <width>281</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Supprimer</string>
      </property>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>53</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date debut :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>24</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="id_supp_ticket">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="nom_supp_ticket">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="prix_supp_ticket">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="datedeb_supp_ticket">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="supp_ticket">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>180</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_55">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>144</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre d'heure  :</string>
       </property>
      </widget>
      <widget class="QTextBrowser" name="nbheure_supp_ticket">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>230</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Tri Par :</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_ticket">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>791</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter</string>
      </property>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_21">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>103</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Date debut :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_ajout_ticket">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix_ajout_ticket">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="ajout_ticket">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QDateEdit" name="datedeb_ajout_ticket">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_53">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre d'heure  :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbheure_ajout_ticket">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>146</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>281</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QPushButton" name="modif_ticket">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>180</y>
         <width>75</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nom_modif_ticket">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_22">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Prix :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="prix_modif_ticket">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>78</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_23">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>53</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date debut :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>24</y>
         <width>47</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Id :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="id_modif_ticket">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="datedeb_modif_ticket">
       <property name="geometry">
        <rect>
         <x>112</x>
         <y>110</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="nbheure_modif_ticket">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>146</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_54">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Nombre d'heure  :</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>230</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_tri_ticket">
      <property name="geometry">
       <rect>
        <x>671</x>
        <y>230</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
